agent/app/agent_capability.go

```diff
@@ -63,7 +64,11 @@ func (agent *ecsAgent) capabilities() []*ecs.Attribute {
 	// Determine API versions to report as supported. Supported versions are also used for capability-enablement, except
 	// logging drivers.
 	for _, version := range agent.dockerClient.SupportedVersions() {
-		capabilities = appendNameOnlyAttribute(capabilities, capabilityPrefix+"docker-remote-api."+string(version))
+		if dockerLegacyCapabilityPrefix[string(version)] {
```

Comment:
This feels like it will be confusing from an administrative perspective. Can we insist that we keep all docker version capabilities using the old style for attributes?